proto/buf/alpha/registry/v1alpha1/studio_request.proto

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// StudioRequestService exposes the functionality to manage favorited Requests
+// from Studio.
+service StudioRequestService {
+  // Create registers a favorite Studio requests to the caller's BSR profile.
+  rpc Create(CreateRequest) returns (CreateResponse) {}
+  // Delete removes a favorite Studio request from the caller's BSR profile.
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  // List shows the caller's favorited requests.
+  rpc List(ListRequest) returns (ListResponse) {}
```

Comment:
yes, scoping the naming as such is consistent with the rest of our APIs - consistency is the most important, and having `CreateRequest/Response` etc means that we're saying "this is the only create type in the entire registry API"

Reply:
You're right. I was trying to avoid stuttering

```
CreateStudioRequestRequest
CreateStudioRequestResponse
```

But since the resource name is `StudioRequest` there's no way out of it.